dali/test/python/test_pipeline.py

```diff
@@ -1104,6 +1109,11 @@ def iter_setup(self):
             break
     assert(iter_num == i)
 
+    iterator = iter(TestIterator(iter_num))
+    pipe = IterSetupPipeline(iterator, 3, 0, True)
+    pipe.build()
+    assert_raises(RuntimeError, pipe.run)
```

Comment:
seems that this belongs to `test_external_source_fail` case

Reply:
done